docs/compilation.rst

```diff
@@ -26,18 +26,21 @@ Building Python wheel and (optionally) Docker image
 
 Change directory (``cd``) into ``docker`` directory and run ``./build.sh``. If needed, set the following environment variables:
 
-* PYVER - Python version. Default is ``3.6``.
-* CUDA_VERSION - CUDA toolkit version (9 for 9.0 or 10 for 10.0). Default is ``10``. If the version is prefixed with `.` then any value ``XX`` can be passed and the user needs to make sure that Dockerfile.cudaXX.deps is present in `docker/` directory.
-* NVIDIA_BUILD_ID - Custom ID of the build. Default is ``1234``.
-* CREATE_WHL - Create a standalone wheel. Default is ``YES``.
-* BUILD_TF_PLUGIN - Create a DALI TensorFlow plugin as well. Default is ``NO``.
+* PYVER - Python version. The default is ``3.6``.
+* CUDA_VERSION - CUDA toolkit version (9 for 9.0 or 10 for 10.0). The default is ``10``. If the version is prefixed with `.` then any value ``XX`` can be passed and the user needs to make sure that Dockerfile.cudaXX.deps is present in `docker/` directory.
+* NVIDIA_BUILD_ID - Custom ID of the build. The default is ``1234``.
+* CREATE_WHL - Create a standalone wheel. The default is ``YES``.
+* BUILD_TF_PLUGIN - Create a DALI TensorFlow plugin wheel as well. The default is ``NO``.
+* PREBUILD_TF_PLUGINS - If prebuild DALI TensorFlow plugin. It should be used together with BUILD_TF_PLUGIN option. If both options are set to ``YES`` then DALI TensorFlow plugin package is built with prebuild plugin binaries inside. If PREBUILD_TF_PLUGINS is set to ``NO`` then the wheel is still built but not prebuild binaries are placed inside and the user needs to make sure that he has proper compiler version present (aligned with the one used to build present TensorFlow) so the plugin can be built during the installation of DALI TensorFlow plugin package. If is BUILD_TF_PLUGIN is set to ``NO`` PREBUILD_TF_PLUGINS value is disregarded. The default is ``YES``.
```

Comment:
```suggestion
* PREBUILD_TF_PLUGINS - Whether to prebuild DALI TensorFlow plugin. It should be used together with BUILD_TF_PLUGIN option. If both options are set to ``YES`` then DALI TensorFlow plugin package is built with prebuilt plugin binaries inside. If PREBUILD_TF_PLUGINS is set to ``NO`` then the wheel is still built but without prebuilding binaries - no prebuilt binaries are placed inside and the user needs to make sure that he has proper compiler version present (aligned with the one used to build present TensorFlow) so the plugin can be built during the installation of DALI TensorFlow plugin package. If is BUILD_TF_PLUGIN is set to ``NO`` PREBUILD_TF_PLUGINS value is disregarded. The default is ``YES``.
```

Reply:
The diff doesn't color this well, but I would change some [pre]build into past participle form [pre]built.

Reply:
Done in https://github.com/NVIDIA/DALI/pull/1407.